### PR TITLE
Add supported operating systems note

### PR DESCRIPTION
**Pull request by dlukeomalley**

### CODE REVIEW

File: docs/intro/installation.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@ Installation
 Requirements
 ------------
 
+.. note:: We currently support Ubuntu and Mac OSX for running ``r2c`` and developing analyzers. While not officially supported, some users have used Windows Subsystem for Linux (WSL) on Windows 10.
```
Comment (ievans) [inline docs/intro/installation.rst]: @dlukeomalley might want to specify versions, esp. for Ubuntu

Comment (dlukeomalley) [reply docs/intro/installation.rst]: Do you now those versions off-hand?

Comment (dlukeomalley) [inline docs/intro/installation.rst]:
```suggestion
.. note:: We currently support > Ubuntu Trusty (14.04.6) and > mac OS Mohave for running ``r2c`` and developing analyzers. While not officially supported, some users have used Windows Subsystem for Linux (WSL) on Windows 10.
```

Comment (dlukeomalley) [reply docs/intro/installation.rst]: CC @ievans @ulziibay

Comment (ulziibay) [reply docs/intro/installation.rst]: It is going to invite questions around what other versions of Ubuntu, mac OS we support. Also, we should let folks know that we are interested in hearing which platform you would like to use R2C tools on

**Pull request merged**